pkg/sentry/sentry.go

```diff
@@ -0,0 +1,30 @@
+package sentry
+
+import (
+	raven "github.com/getsentry/raven-go"
+)
+
+// Capture reports an error message to sentry along with a stacktrace.
+//
+// skipFrames can be used to indicate how many stack frames should be skipped to
+// reach the frame where the error actually happened instead of always pointing
+// to a generic "logError()".
+//
+// tags are additional tags to add the sentry event.
+//
+// This call is synchronous in that it will not return until the message has
+// been sent to sentry.
+func Capture(message string, skipFrames int, tags map[string]string) {
+	// We want to at least skip the Capture() function itself.
+	if skipFrames < 1 {
```

Comment:
Shouldn't this consistently `+= 1` to skipFrames? That way the caller only has to count the number of frames they want to skip.